Doc/library/email.parser.rst

```diff
@@ -265,6 +265,9 @@ in the top-level :mod:`email` package namespace.
    ``Parser().parsestr(s)``.  *_class* and *policy* are interpreted as
    with the :class:`~email.parser.Parser` class constructor.
 
+   To get a decoded message, set policy as 'email.policy.default'. For
```

Comment:
Maybe this should say "to decode message headers" instead of "to get a decoded message".

Reply:
Thanks for your comments. Its updated. 

The link is provided to the policy instead of BytesFeedParser as it has details on different policy options. Hope that is fine.